app/assets/stylesheets/custom.css.scss

```diff
@@ -6,6 +6,12 @@
 $gray-medium-light: #eaeaea;
 $white: #ffffff;
 
+@mixin box_sizing {
```

Comment:
Name of mixin `box_sizing` should be written in all lowercase letters with hyphens instead of underscores